server/command.go

```diff
@@ -108,7 +108,7 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User) (str
 	}
 
 	if recentMeeting {
-		p.postConfirm(recentMeetingLink, args.ChannelId, "", user.Id, creatorName, provider)
+		p.postConfirm(recentMeetingLink, args.ChannelId, "", user.Id, args.RootId, args.ParentId, creatorName, provider)
```

Comment:
You can forget about the ParentID. It was not being used for a while, and has been deprecated in the latest versions of the server.

Reply:
I was sceptical about the same, since both variable hold same value and they both exist in docs with no explanation. Tried with individual for each, didn't work for me. Will retry.

Reply:
ParentID is the leftovers from a long time ago when they tried to have several levels of threading (threads inside threads inside threads). That idea was scrapped long ago, but the ParentID value kept lingering until now, that has been removed too. You should be able to do this ignoring completely the ParentID.

Reply:
All right, will remove it.